client/src/App.jsx

```diff
@@ -1,17 +1,35 @@
 import { Route, Routes, NavLink } from 'react-router-dom';
+import { useState } from 'react';
 import Navbar from './components/Navbar';
 import Home from './pages/Home';
 import Create from './pages/Create';
 import Settings from './pages/Settings';
 import styled from 'styled-components';
+import { logData } from './components/LogData';
 
 function App() {
+  const [logs, setLogs] = useState(logData);
+
+  function AddLog(log) {
```

Comment:
I'd recommend to name this function in lower case, since upper case is usually used for components in React.
```suggestion
  function addLog(log) {
```